src/routes/eventRouter.js

```diff
@@ -0,0 +1,54 @@
+//dependencies
+const routes = require('express').Router();
+const data = require('../../exampleData.json');
+const { Event } = require('../database/models/eventsModel');
+const { getDatabase } = require('../database/mongo');
+
+// GET JSON of all events
+routes.get('/all', async (req, res, next) => {
+
+    Event.find(function (err, messages) {
+        if (err) {
+          console.log('error ', err.res);
+        }
+        else {
+          console.log(messages);
+          res.status(200).json(messages);
+        }
+      })
+});
+
+
+// POST a new event to the database
+routes.post('/event', (request, res) => {
+    console.log(request.body);
+    if (!request.body || request.body == {}){
+      return res.send("no request body")
+    }
+    var event = new Event({
+        title: request.body.title,
+        description: request.body.description,
```

Comment:
Gonna tag @Alacau here - the design sould match whatever is sent in the POST. Not sure if all the form componens are done

Reply:
event title, event date, event location, start time, end time, check-in code, background image

what's missing here is event date in this format (January 10, 2020) and event location
still need to confirm with @leestanley if we need a description

Reply:
yea for sure, its flexible to change the schema/model based on what we need (so i can remove description if we dont need it).

The start time/end DATE type include the date and time in format ("2020-04-25T18:00:00Z" for April 25, 2020 at 18:00 UTC time) but we can move to a different format if its easier. Haven't worked with time zones much so I'm not sure if its easy to convert UTC to local timezone or not.

Reply:
@Alacau Description doesn't seem necessary for now. The front end should send it as a JavaScript date type.